src/modules/otc.ts

```diff
@@ -0,0 +1,61 @@
+import { MsgAcceptQuote, MsgCancelRfq, MsgCreateRfq } from "@carbon-sdk/codec/Switcheo/carbon/otc/tx";
+import { CarbonTx } from "@carbon-sdk/util";
+import BaseModule from "./base";
+
+export class OTCModule extends BaseModule {
+  public async createRfq(params: OTCModule.CreateRfqParams, opts?: CarbonTx.SignTxOpts) {
+    const wallet = this.getWallet();
+    const { denoms: sellCoins, buyDenom, expiryTime } = params;
+
+    const value = MsgCreateRfq.fromPartial({
+      requester: wallet.bech32Address,
+      sellCoins,
+      buyDenom: buyDenom ?? 'swth',
+      expiryTime: expiryTime ?? new Date(0),
```

Comment:
should be can remove both fallbacks since `buyDenom` and `expiryTime` are required params and wont be undefined